src/neo/Consensus/ConsensusService.cs

```diff
@@ -30,6 +30,8 @@ internal class Timer { public uint Height; public byte ViewNumber; }
         private DateTime block_received_time;
         private bool started = false;
 
+        private static uint furtherPayloadToSave = 10;
```

Comment:
I think this could be `1`. Because if we lag behind alot, we should use the block synchronization mechanism.

Reply:
We can reduce to `2`, `3`, but `1` is almost nothing, @erikzhang.
Let's make it as a function of BlockTime, then, on privatenets this number will be bigger.

Reply:
Modified to 3, @erikzhang, I think it might be enough for now.
In the future we may discuss a more dynamic metric.